libnavigation-base/src/main/java/com/mapbox/navigation/base/internal/CurrentIndicesSnapshotFactory.kt

```diff
@@ -0,0 +1,14 @@
+package com.mapbox.navigation.base.internal
+
+import com.mapbox.navigation.base.CurrentIndicesSnapshot
+
+object CurrentIndicesSnapshotFactory {
+
+    fun createIndices(
+        legIndex: Int = 0,
+        routeGeometryIndex: Int = 0,
+        legGeometryIndex: Int? = null,
```

Comment:
Why do these have default values? They would nearly always be incorrect. Can we force the caller to provide the values instead?